docling_core/types/doc/base.py

```diff
@@ -81,6 +83,20 @@ def normalized(self, page_size: Size) -> "BoundingBox":
 
         return out_bbox
 
+    def scale_to_size(self, page_size: Size) -> "BoundingBox":
```

Comment:
The `Size` argument is used to scale with different factors in each direction. I don't think the name `page_size` is correct here, because it has (almost) nothing to do with the page.

What about `size`? or `scales`? or `multipliers`?